server/channels/app/support_packet.go

```diff
@@ -288,3 +291,18 @@ func (a *App) createSanitizedConfigFile() (*model.FileData, error) {
 	}
 	return fileData, nil
 }
+
+func (a *App) createHeapProfile() (*model.FileData, error) {
```

Comment:
This needs to be done across all nodes in a cluster.

Reply:
You are correct, the current implementation only captures the pprof information at a single node.

That is by design. Adding the pprof information to the support package is a test to see how useful this information is and if it reduces round trip times between support and customer. It would surely be nice to have the information from all nodes, but I don't see it as a requirment. Please note that there is also the workaround of fetching the support package from all nodes individually, if one really needs the information from all nodes.

Reply:
cc @Willyfrog as we talked about this offline

Reply:
IMO, a heap profile from a random node is useful when we just want to take a look at memory usage in normal conditions. But it's not very useful when there's an actual problem, because there might be only one node with a memory leak and there's no guarantee which node this request will hit.

> Please note that there is also the workaround of fetching the support package from all nodes individually, if one really needs the information from all nodes.

I would say in that case, one might as well manually grab heap profile from all nodes as well.

I understand that we are merely trying to cram as much information as possible in the support packet, but in times of an incident or an outage, having a profile from only one node isn't terribly useful. So one would eventually have to grab profiles from all nodes anyways.